t/python/t0013-job-list-inactive.py

```diff
@@ -238,4 +235,4 @@ def test_09_middle_timestamp_2(self):
     if rerun_under_flux(size=__flux_size(), personality="job"):
         from pycotap import TAPTestRunner
 
-        unittest.main(testRunner=TAPTestRunner())
+        unittest.main(testRunner=TAPTestRunner(), verbosity=2)
```

Comment:
Ugh. Forgot to hit `start a review` before approving.  Sorry.

Do we want to leave this verbosity up permanently?  If not, maybe we open an issue about tying to the `debug` env var much like `FLUX_PYCLI_LOGLEVEL` is in #2804 .